### PR TITLE
Makes antitamper mechanisms go off when the crate is tampered with.

### PR DESCRIPTION
**Pull request by Screemonster**
:cl:
tweak: The anti-tamper mechanism on abandoned crates now resists tampering.
/:cl:

Adds a 2% chance per shot of bullet_act making the anti-tamper mechanism on abandoned crates blow up. With a 20 damage weapon and secure crates having 1000 health, this works out as a 64% chance of exploding before you crack it open. This drops to 50% with a 30 damage weapon.

Adds a 90% chance of explosions setting it off as well.

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
BUT HOW AM I GOING TO CRACK OPEN CRATES NOW?

**Comment by Screemonster** [comment]
Solve the puzzle you lazy bum :v

In all seriousness, the easy methods to bypass the puzzle entirely are the number one reason _not_ to put better loot in them.
I considered moving boom() up to secure crates in general along with a new var to set the chance of exploding, but that's something I'd wanna talk to people about before _royally shitting on guncargo_

File: code/modules/mining/abandoned_crates.dm [hunk 1]
```diff
@@ -187,6 +187,17 @@
 		return
 	attack_hand(user)
 
+/obj/structure/closet/crate/secure/loot/bullet_act(obj/item/projectile/P)
+	if(prob(2)) //parent has 1000 health, 20 damage weapon has 36% chance of getting the loot.
+		boom()
```
Comment (phil235) [inline code/modules/mining/abandoned_crates.dm]: shoot the crate with a taser, or even a lasertag laser : BOOM

**Comment by Iamgoofball** [comment]
:-1: make them guaranteed to trigger the anti tamp er

File: code/modules/mining/abandoned_crates.dm (hunk 1 above)
Comment (Jordie0608) [inline code/modules/mining/abandoned_crates.dm]: Or a wand of healing.

Comment (Screemonster) [inline code/modules/mining/abandoned_crates.dm]: I can put the same "if burn or brute" check that depleting the health does if that's preferred.

**Comment by Screemonster** [comment]
#20793 completely overhauls the way crates take damage so I'm doing a little rewrite.

**Pull request closed without merging**